app.js

```diff
@@ -2,9 +2,24 @@ const express = require("express");
 const app = express();
 
 const getApi = require("./controllers/api.controller");
+const { getTopics } = require("./controllers/topics.controller");
+const {
+  psqlErrorHandler,
+  customErrorHandler,
+  serverErrorHandler,
+} = require("./error-handlers");
 
 app.use(express.json());
```

Comment:
Is this required at this point? Following TDD and only writing code that is necessary to pass your tests adding it in at this point before building the feature seems a little redundant